src/data/sampleBookings.js

```diff
@@ -0,0 +1,53 @@
+const sampleBookings = [
```

Comment:
In our meeting, Nik mentioned that the roomServiceCharges property were NOT something we needed to worry about for this project and is consequently something we don't need to test for. So I didn't. I suppose I could've just deleted them since she said she was going to update the api. Put it on the to do list!